php/classes/handlers/class-castos-handler.php

```diff
@@ -362,15 +372,56 @@ public function upload_podcast_to_podmotor( $post ) {
 	}
 
 	/**
-	 * Gets the featured image url
+	 * Gets podcast image url
 	 *
 	 * @param $post
-	 * @param $post_body
 	 *
-	 * @return mixed
+	 * @return string
+	 */
+	public function get_podcast_image_url( $post ) {
```

Comment:
I'm a bit pedantic about naming, as the meta key for this data is called `cover_image` could we rename this method to get_cover_image_url. And then any variables that are assigned elsewhere from this method are updated accordingly.

This is to ensure that someone does not confuse this with the actual featured image, as determined by the regular WordPress functionality.

Reply:
@jonathanbossenger Changed it.